templates/vue/src/App.vue

```diff
@@ -22,37 +22,45 @@ export default {
 
 html {
   font-size: 100%;
+}
```

Comment:
It seems like this is reverting the SASS changes. Could you please make sure this is written is SCSS instead and not revert that change?

Reply:
Sorry about that, looks like a problem from doing the merge. Fixed.